app/src/main/java/fr/free/nrw/commons/settings/SettingsFragment.java

```diff
@@ -144,10 +141,42 @@ public void afterTextChanged(Editable s) {
         }
     }
 
+    /**
+     * Uses previously saved theme if there is any, if not then uses default.
+     * Adds preference theme listener and saves value chosen by user to shared preferences
+     * to remember later
+     */
+    private void prepareTheme() {
+
+        String currentTheme = getCurrentTheme();
+        if (currentTheme.equals("")){
```

Comment:
this is never true

Reply:
I understand now, thanks for pointing it out, I removed it!